aten/src/ATen/native/cuda/ReduceOpsKernel.cu

```diff
@@ -60,7 +60,7 @@ void mean_kernel_impl(TensorIterator& iter) {
 template <typename scalar_t, typename acc_t=scalar_t, typename out_t=scalar_t>
 void norm_kernel_cuda_impl(TensorIterator& iter, Scalar val) {
   float p;
-  if (val.isIntegral()) {
+  if (val.isIntegral(true)) {
```

Comment:
any particular reason for this?  It seems slightly nicer to just make this an error and we can turn it on if people ask.